Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/outbound/StandardOutboundPatientDiscovery.java

```diff
@@ -410,7 +410,7 @@ protected PatientDiscoveryAuditLogger getNewPatientDiscoveryAuditLogger() {
     }
 
     private void auditRequest(PRPAIN201305UV02 request, AssertionType assertion, NhinTargetSystemType target) {
-        patientDiscoveryAuditor.auditRequestMessage(request, assertion, target,
+        getNewPatientDiscoveryAuditLogger().auditRequestMessage(request, assertion, target,
```

Comment:
Don't we call this "getAuditLogger" for the other services?  Or am I thinking of a different function?